src/org/omegat/core/machinetranslators/ApertiumTranslate.java

```diff
@@ -123,6 +143,9 @@ protected String getJsonResults(String json) {
         if (rootNode.containsKey("responseStatus")) {
             code = (Integer) rootNode.get("responseStatus");
         }
+        else {
```

Comment:
Our coding style would put this `else` on the previous line with the closing bracket. To avoid having to nitpick these things, I made a Gradle task to auto-format files changed on the current branch:

```
./gradle spotlessChangedApply
```

Could you run it and see if it works for you?

Reply:
Thanks, I did not know about this and it has detected other things in addition to this. Fixed in the latest commit.